docs/_guide/lifecycle.md

```diff
@@ -279,6 +279,19 @@ The `updateComplete` Promise resolves when the element has finished updating. Us
   this.updateComplete.then(() => { /* do stuff */ });
   ```
 
+To await additional state before fulfilling the `updateComplete` promise, override the `_getUpdateComplete` method. For example, it may be useful to await the update of a child element here. First await `super._getUpdateComplete()`, then any subsequent state.
```

Comment:
If overriding `_getUpdateComplete` is less common than using `this.updateComplete`, I think this note should be _after_ the main example.

I'd suggest moving this after the `updateComplete` example, and introducing it with a L4 heading:

`#### Overriding updateComplete`

(to be clear, I'm suggesting inserting it just before `## Examples`, ~L302 in this PR.

Reply:
SGTM, done!